docs/UserGuide.adoc

```diff
@@ -239,29 +239,28 @@ Format: `edit INDEX [n/EXPENSE_NAME] [$/COST] [c/CATEGORY] [d/DATE] [t/TAG...]`
 Edits the expenses with the keywords specified by the user.
 Existing values of the expense will be edited according to the value of the parameters +
 
-Format: `massedit [n/EXPENSE_NAME] [$/COST] [c/CATEGORY] [d/DATE] [t/TAG...] -> [n/EXPENSE_NAME] [$/COST] [c/CATEGORY] [d/DATE] [t/TAG...]`
+Format: `massedit [n/EXPENSE_NAME] [$/COST] [c/CATEGORY] [d/DATE] [t/TAG...] \-> [n/EXPENSE_NAME] [$/COST] [c/CATEGORY] [d/DATE] [t/TAG...]`
 
 ****
 *Examples:*
 
-* `massedit c/school -> c/food` +
+* `massedit c/school \-> c/food` +
```

Comment:
Is the slash supposed to be there

Reply:
Yes, this is to ensure that the "->" will not become "→" automatically in UserGuide.